packages/playground/src/components/playground/PlaygroundPageContent/PlaygroundPageContent.tsx

```diff
@@ -38,108 +27,11 @@ export const PlaygroundPageContent: React.FC = (_props) => {
 
   const formRef = useRef<HTMLFormElement>(null);
 
-  useHotkeys(
-    "mod+enter",
-    () => {
-      if (formRef.current) {
-        formRef.current.requestSubmit();
-      }
-    },
-    {
-      enableOnFormTags: ["input"],
-    },
-  );
-
   const isLgScreen = useIsLgScreen();
 
-  const [commandBarOpen, setCommandBarOpen] = useState(false);
-
-  useHotkeys(
-    "mod+k",
-    (e) => {
-      e.preventDefault();
-      setCommandBarOpen(true);
-    },
-    {
-      enableOnFormTags: ["input"],
-    },
-  );
-
-  useHotkeys(
-    "mod+/",
-    () => {
-      setShortcutsOpen(true);
-    },
-    {
-      enableOnFormTags: ["input"],
-    },
-  );
-
-  useKeySequence(
-    ["g", "d"],
-    () => {
-      if (visibleRequestsPanelTabs.includes("docs")) {
-        setActiveRequestsPanelTab("docs");
-      }
-    },
-    { description: "View Route Docs", ignoreSelector: "[contenteditable]" },
-  );
-
-  useKeySequence(
-    ["g", "p"],
-    () => {
-      if (visibleRequestsPanelTabs.includes("params")) {
-        setActiveRequestsPanelTab("params");
-      }
-    },
-    { description: "View Request Params", ignoreSelector: "[contenteditable]" },
-  );
-
-  useKeySequence(
-    ["g", "a"],
-    () => {
-      if (visibleRequestsPanelTabs.includes("auth")) {
-        setActiveRequestsPanelTab("auth");
-      }
-    },
-    {
-      description: "View Request Auth",
-      ignoreSelector: "[contenteditable]",
-    },
-  );
-
-  useKeySequence(
-    ["g", "h"],
-    () => {
-      if (visibleRequestsPanelTabs.includes("headers")) {
-        setActiveRequestsPanelTab("headers");
-      }
-    },
-    {
-      description: "View Request Headers",
-      ignoreSelector: "[contenteditable]",
-    },
-  );
-
-  useKeySequence(
-    ["g", "b"],
-    () => {
-      // TODO - Focus the body input after this
-      if (visibleRequestsPanelTabs.includes("body")) {
-        setActiveRequestsPanelTab("body");
-      }
-    },
-    { description: "View Request Body", ignoreSelector: "[contenteditable]" },
-  );
-
-  useKeySequence(
-    ["g", "d"],
-    () => {
-      if (visibleRequestsPanelTabs.includes("docs")) {
-        setActiveRequestsPanelTab("docs");
-      }
-    },
-    { description: "View Route Docs", ignoreSelector: "[contenteditable]" },
+  const { commandBarOpen, setCommandBarOpen } = useStudioStore(
```

Comment:
okay another thing: the `<CommandBar` component is still only rendered on this page, so we may need to move it up into the layout component itself, so that each page still has the component that can render the command bar itself